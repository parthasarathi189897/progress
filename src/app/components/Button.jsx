import React from 'react';

const Button = (props) => {
	const _butClick = () => {
		props.butClick(props.butVal);
	};
	return (
		<button type='submit' onClick={_butClick}>{props.butVal}</button>
	);
};

export default Button;