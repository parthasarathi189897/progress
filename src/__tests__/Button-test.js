import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../app/components/Button';


it('renders correctly', () => {
  var myMock = jest.fn();
  const tree = renderer.create(
    <Button buttonClick={myMock} butVal="5" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

