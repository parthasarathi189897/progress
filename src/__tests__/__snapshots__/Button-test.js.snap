// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="progress-button"
  onClick={[Function]}
  type="submit"
>
  5
</button>
`;
